webete/webete.py

```diff
@@ -23,21 +25,35 @@ def start_section(name):
 
 def action_auto(target):
     start_section('robots.txt')
-    r = requests.get(target + 'robots.txt')
+    r = requests.get(target + 'robots.txt')  # does not need auth (it would be stupid)
     if r.status_code == 200:
         print(r.text)
     else:
         print(r.status_code)
 
 
-def action_python(target, fpath):
+def action_python(target, fpath, ausr, apwd):
     # Should we support these?
     # foo.cpython-35.opt-1.pyc
     # foo.cpython-35.opt-2.pyc
     # pypy?
     PYTHON_EXTS = ['pyc', 'pyd', 'pyo']
     PYTHON_VERSIONS = ['26', '27', '35', '36', '37']
 
+    # Basic Auth Block. In the future this should go in another and specialized function
+    http_auth = None
+
+    log.debug('Http Basic auth: {}'.format((ausr, apwd)))
+    if(ausr != apwd) and (ausr is None or apwd is None):
```

Comment:
Needs space in the first parenthesis or just remove them.
Also, the logic condition is a pain to understand. With a separated `--basic-auth-whatever` flag you can avoid these checks.